docs/macros-ref.rst

```diff
@@ -88,7 +88,12 @@ attributes and methods.
 The ``var`` template variable allows you to access variables defined in Airflow's
 UI. You can access them as either plain-text or JSON. If you use JSON, you are
 also able to walk nested structures, such as dictionaries like:
-``{{ var.json.my_dict_var.key1 }}``
+``{{ var.json.my_dict_var.key1 }}``.
+
+It is also possible to fetch a variable by string if needed with
+``{{ var.value.get('my.var', 'fallback') }}`` or
```

Comment:
Instead of needing a code change can we instead use Jinja's built in `default` fliter? `{{ var.value['my.var'] | default 'fallback' }}`?

https://jinja.palletsprojects.com/en/2.10.x/templates/#default

Reply:
This resolves my original reason for adding this method (periods in variable keys), but that default logic doesn't work because it throws a KeyError first when it's missing. `Error rendering template: 'Variable my.var does not exist'` is the error I get from your example.

Reply:
Ah okay, yeah I thought Jinja might catpure a KeyError and handle it for the default filter, but I guess not. Cool, just wanted to check if this PR was needed, and it looks like it is. :+1: